### ISSUE TITLE
overlapping tables in same level with LCS after ENOSPC

### ISSUE THREAD

**Issue by gleb-cloudius** [scylladb/scylla#2416]
*Installation details*
Scylla version (or git commit hash): 1.7

Restarting node after ENOSPC prints a lot of those:

```
May 25 03:37:14 n1.sahara.cloudius-systems.com scylla[56862]:  [shard 10] LeveledManifest - At level 3, /var/lib/scylla/data/outbrain_document_features/document_features-12e054103bcf11e7af5500000000000a/outbrain_document_features-document_features-ka-115114-Data.db [pk{0008000000005ae03afe}, pk{0008000000010f4b6a57}] overlaps /var/lib/scylla/data/outbrain_document_features/document_features-12e054103bcf11e7af5500000000000a/outbrain_document_features-document_features-ka-90058-Data.db [pk{0008000007facf2573e4}, pk{000800000001dfd93a13}]. This could be caused by the fact that you have dropped sstables from another node into the data directory. Sending back to L0.
```

I did not dropped  sstables from  another node.

**Comment by slivne** [comment]
@raphaelsc seems related to #1807

**Comment by raphaelsc** [comment]
On Sun, May 28, 2017 at 9:48 AM, Shlomi Livne <notifications@github.com>
wrote:

> @raphaelsc <https://github.com/raphaelsc> seems related to #1807
> <https://github.com/scylladb/scylla/issues/1807>
>

exactly, this issue seen by Gleb is an symptom of #1807 not fixed.

**Comment by avikivity** [comment]
Fixed by d82a8dfae0f184f7dae1c710732c07144974851f.
